### DISCUSSION TITLE
Is there a way to see old docs?

### DISCUSSION THREAD

**Discussion by cadop** [NVIDIA/warp#167] [Q&A] [answered]
The docs seem to be on the beta release, but when using `pip install warp-lang` its on `10.1`, so some of the docs don't seem to apply

**Comment by shi-eric** [comment, marked as answer]
Good question. Here are two ways to get docs for a different version:

- Clone the repo, check out the version you want using the appropriate tag, then build the docs using `python build_docs.py`. The HTML documentation output will be in `docs/_build/html/`
- From the [tags page](https://github.com/NVIDIA/warp/tags) on Github, choose your desired version. Download the source code archive, which when unzipped has the documentation in `docs/_build/html/`
